src/base/system_parameter.c

```diff
@@ -705,6 +705,7 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 #define PRM_NAME_FLASHBACK_TIMEOUT "flashback_timeout"
 #define PRM_NAME_FLASHBACK_MAX_TRANSACTION "flashback_max_transaction"
 #define PRM_NAME_FLASHBACK_WIN_SIZE "flashback_win_size"
+#define PRM_NAME_HOSTS_CONF "hosts_conf"
```

Comment:
How about **use_user_hosts** as the parameter name?

Reply:
It looks so good. I will apply it.